pkg/userdata/userdata.go

```diff
@@ -121,11 +121,12 @@ type File struct {
 
 // Install represents the installation options for preparing a node.
 type Install struct {
-	Boot  *BootDevice    `yaml:"boot,omitempty"`
-	Root  *RootDevice    `yaml:"root"`
-	Data  *InstallDevice `yaml:"data,omitempty"`
-	Wipe  bool           `yaml:"wipe"`
-	Force bool           `yaml:"force"`
+	Boot         *BootDevice      `yaml:"boot,omitempty"`
+	Root         *RootDevice      `yaml:"root"`
+	Data         *InstallDevice   `yaml:"data,omitempty"`
+	ExtraDevices []*InstallDevice `yaml:"extraDevices,omitempty"`
```

Comment:
should we allow mount points to be configured as part of the extra devices?

Reply:
Good point. We absolutely should. Otherwise they have no way to handle that. Good catch.